engine/crates/engine/src/registry/resolvers/mod.rs

```diff
@@ -235,11 +265,24 @@ impl Resolver {
                         variable_definitions,
                         registry,
                         Some(batcher),
-                    )
+                    );
+
+                #[cfg(feature = "tracing_worker")]
+                let future = future.instrument(info_span!("http_resolver"));
```

Comment:
```suggestion
                let future = future.instrument(info_span!("graphql_resolver"));
```

Could probably at least put `resolver.id_or_name` in as field here as well?